rllib/utils/replay_buffers/episode_replay_buffer.py

```diff
@@ -537,6 +550,8 @@ def _sample_episodes(
 
             # Skip, if we are too far to the end and `episode_ts` + n_step would go
             # beyond the episode's end.
+            if burnin > 0 and episode_ts + burnin >= len(episode):
```

Comment:
not needed?

Reply:
This is the place where my comment above would point to. We check here, if we have a burnin and guarantee that the sampled sequence is at least as long as the required burn-in b/c otherwise the full mask would be `False` and we would have sampled for no update.